src/store/RelayRecordStore.js

```diff
@@ -124,11 +126,12 @@ class RelayRecordStore {
         'cannot be null or undefined.',
         storageKey
       );
-      return identifyingArgValue;
+      return stringifyArg(identifyingArgValue);
```

Comment:
This section of code is a special case for the `Node` interface: a field that you can use to fetch a record by its global ID. We want to leave this special case intact; you should never be able to use `node(id: …)` with anything other than an `id` argument of type `ID!`. See https://facebook.github.io/relay/graphql/objectidentification.htm for more information.